drafting.typ

```diff
@@ -398,7 +513,7 @@
 /// - dy (length): Vertical offset from the note's location -- negative values
 ///   move the note up, positive values move the note down
 /// - ..kwargs (dictionary): Additional properties to apply to the note. Accepted values are keys from `margin-note-defaults`.
-#let margin-note(body, dy: auto, ..kwargs) = {
+#let margin-note(body, dy: auto, ..kwargs) = [ #h(0pt) #{ // h(0pt) forces here().position() to take paragraph indent into account
```

Comment:
I'm not really happy about this approach btw, if you have a suggestion...

But setting
```typ
#set par(first-line-indent: 1em)
```

Results in notes in a new paragraph not being place properly